### ISSUE TITLE
Verständnisfrage

### ISSUE THREAD

**Issue by spymo** [kogpsy/neuroscicomplab#22]
Es gilt:
P(θ|Data)∝P(Data|θ)∗P(θ) 

das bedeutet, wenn in meinem Prior eine  "0" vorkommt, wie hier:

prior2 <- ifelse(p_grid < 0.5 , 0 , 1)
compute_posterior(likelihood, prior2)

wird mein posterior für diese Werte auch immer "0" sein, da es ja eine Multiplikation ist, oder? Ist es nicht komisch eine Position einzunehmen, von der man niemals durch Beobachtung abgebracht werden kann? Wahrscheinlich diskutierst du das eh später. Ich lese mal weiter.

**Comment by awellis** [comment]
Ja, das ist richtig. Wenn du apriori davon überzeugt bist, dass die Wahrscheinlichkeitsdichte 0 ist, dann kann nichts deine Meinung ändern.

Könntest du ausführlicher beschreiben, was du mit komisch meinst? Ich verstehe, worauf du hinaus willst; unter einer Betaverteilung hat der ganze Wertebereich zwischen 0 und 1 eine nonzero Dichte, aber aufgrund von welcher Argumentation behauptest du, dass eine bestimmte apriori Überzeugung flaschoder komisch ist?

**Comment by awellis** [comment]
Solche Fragen kannst im `Discussions` Teil stellen: [https://github.com/kogpsy/neuroscicomplab/discussions](https://github.com/kogpsy/neuroscicomplab/discussions).
